python/cudf/cudf/_lib/cpp/copying.pxd

```diff
@@ -175,10 +175,3 @@ cdef extern from "cudf/copying.hpp" namespace "cudf" nogil:
     ctypedef enum sample_with_replacement:
         FALSE 'cudf::sample_with_replacement::FALSE',
         TRUE 'cudf::sample_with_replacement::TRUE',
-
-    cdef unique_ptr[table] sample (
```

Comment:
Can you raise an issue about removing the libcudf API?

Reply:
#10361